Source/CloudyWebAPI/Private/CloudyWebAPI.cpp

```diff
@@ -405,8 +420,16 @@ void CloudyWebAPIImpl::ReadAndStoreSaveFileURL(FString JsonString, int32 PlayerC
     {
         SaveFileUrls.AddUninitialized(PlayerControllerId - InitialArraySize + 1);
```

Comment:
With our implementation, it will not exceed 3. However I see the bug. I will fix it. Thanks.